src/profile/ChangePassword.js

```diff
@@ -11,91 +11,91 @@ import LoaderButton from "../components/LoaderButton";
  * @TODO refactor into React Hooks
```

Comment:
Would you mind commenting out this @TODO ? It won't be needed once the ticket has been resolved :)

Reply:
Todo comment removed